runtime/syntax/nftables.yaml

```diff
@@ -0,0 +1,20 @@
+filetype: nftables
+
+detect:
+    filename: "nftables.conf$"
+    header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
+    
+rules:
+    - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
+    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
```

Comment:
Could be reduced to `i(cm)?p(x|(v?(4|6))?)`, but I'm unsure if it improves readability or regex compile time.

Reply:
i think not that this can be reduced, because of then would be things like chain, counter, map etc and tcp udp not highlighed.

Reply:
Argh, sure...forget to fill up: `ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp` :man_facepalming:

Reply:
I`ve added your suggestions, you can try if you want to. If i should build it for you, please answer me, then i will build it and create a little release for you. 

But you can clone the Repo, and do the described manual build. (Don`t forget to install golang before)
https://github.com/theredcmdcraft/micro/tree/nftables-improvements

Reply:
I doubt @JoeKar needs help with that. He is a maintainer of micro.

Reply:
> I doubt @JoeKar needs help with that. He is a maintainer of micro.

Oops. I didn`t see the Collaborator Symbol on the right side.

Reply:
No problem, it was my fault not testing it earlier at work were we've some good examples of nftable rules. It was still on my todo before it was merged.
Ok, anyway...since this merge request is already merged we should create a new one to properly add the fixes into the `nftables.yaml`.
I suggest to do the following (after the "[upstream](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository)" is available in your private fork too):
```shell
git check nftables-improvements
git fetch --all
git reset HEAD~1
git stash
git reset --hard upstream/master
git stash apply
git stash add runtime/syntax/nftables.yaml
git commit -m "Added Suggested changes"
git push --force-with-lease
```

Create from this resulting branch (nftables-improvements) in your fork a new pull request to to our upstream repository. Sure you can follow your own approach to create a new clean PR.
Currently I've some doubts that the changed `symbol` and `constant.string` will work as expected.

Reply:
> No problem, it was my fault not testing it earlier at work were we've some good examples of nftable rules. It was still on my todo before it was merged. Ok, anyway...since this merge request is already merged we should create a new one to properly add the fixes into the `nftables.yaml`. I suggest to do the following (after the "[upstream](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository)" is available in your private fork too):
> 
> ```shell
> git check nftables-improvements
> git fetch --all
> git reset HEAD~1
> git stash
> git reset --hard upstream/master
> git stash apply
> git stash add runtime/syntax/nftables.yaml
> git commit -m "Added Suggested changes"
> git push --force-with-lease
> ```
> 
> Create from this resulting branch (nftables-improvements) in your fork a new pull request to to our upstream repository. Sure you can follow your own approach to create a new clean PR. Currently I've some doubts that the changed `symbol` and `constant.string` will work as expected.

Did i have to do these steps, when i synced my repo with this repo? I Synced the Fork, and then i created a new branch from the master branch, where i did the changes yesterday. Or can i simply create a new PR with my newly created branch as source. The Target is the micro master branch.

Reply:
> Did i have to do these steps, when i synced my repo with this repo? I Synced the Fork, and then i created a new branch from the master branch, where i did the changes yesterday.

Sure, you can use a fully new/clean branch derived from the synced master to publish your changes within a new PR.

Reply:
The new PR is #3517